src/services/completions.ts

```diff
@@ -2,10 +2,12 @@
 namespace ts.Completions {
     export enum SortText {
         LocationPriority = "0",
-        SuggestedClassMembers = "1",
-        GlobalsOrKeywords = "2",
-        AutoImportSuggestions = "3",
-        JavascriptIdentifiers = "4"
+        LocationPriorityOptional = "1",

```

Comment:
Renames here as  `LocationPriorityOptional`  to `OptionalMember` and `LocationPriorityFulfilled` to `MemberAleadyDeclared`